app/controllers/stories_controller.rb

```diff
@@ -5,8 +5,9 @@ class StoriesController < ApplicationController
 
   def index
     @all_stories = Story.all
-    @featured_story = @all_stories.first
-    @stories = @all_stories[1..-1]
+    @all_published_stories = @all_stories.reject {|story| story.published == false}
```

Comment:
you can use `@all_stories.published`, rails automatically provides scopes for boolean columns
```
@published_stories = @all_stores.published
```